sn_networking/src/event/mod.rs

```diff
@@ -244,6 +271,11 @@ impl SwarmDriver {
         self.log_kbuckets(&removed_peer);
         self.send_event(NetworkEvent::PeerRemoved(removed_peer, self.peers_in_rt));
 
+        #[cfg(feature = "open-metrics")]
+        if self.metrics_recorder.is_some() {
+            self.check_for_change_in_our_close_group();
```

Comment:
seems there is `log` within this function, and the change not sent to `metrics_recorder` .
is it ok ?